packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -1069,6 +1095,74 @@ describe('<TreeItem />', () => {
     });
 
     describe('Multi Selection', () => {
+      describe('deselection', () => {
+        describe('mouse and multiple nodes are selected', () => {
+          specify('clicking a selected node holding ctrl should deselect the node', () => {
```

Comment:
When do we use `specify` over `it`?

Reply:
I don't think there's a hard rule?

Reply:
I usually use `specify` when `it` doesn't make a readable sentence. Ideally we'd just use `test` if it were available. But I don't see a meaningful problem here.

Reply:
Ok, I was wondering to know when I should be using it.